src/pages/Resume.tsx

```diff
@@ -88,6 +81,57 @@ export default function Resume() {
   );
 }
 
+function useUserInfo() {
+  const [userInfoInput, setUserInfoInput] = useState<UserInfo | null>(null);
+  useQuery({
+    queryKey: ["userInfo"],
+    queryFn: () =>
+      getUser().then((data) => {
+        setUserInfoInput({
+          admission: "",
+          college: data.college,
+          githubId: data.github_email,
+          major: data.department,
+          notionEmail: data.notion_email,
+          slackEmail: data.slack_email,
+          status: "",
+          university: data.university,
+        });
+        return data;
+      }),
+  });
+  return [userInfoInput, setUserInfoInput] as const;
+}
+
+function useSubmit(recruiting_id: number) {
+  const queryClient = useQueryClient();
+  const { mutate } = useMutation(
+    (data: { userInfo: UserInfo; questionaire: ResumeSubmissionCreate[] }) =>
+      Promise.all([
+        patchUser({
+          college: data.userInfo.college,
+          department: data.userInfo.major,
+          github_email: data.userInfo.githubId,
+          notion_email: data.userInfo.notionEmail,
+          slack_email: data.userInfo.slackEmail,
+          university: data.userInfo.university,
+        }),
+        putResume(recruiting_id, data.questionaire),
+      ]),
+    {
+      onSuccess: () => {
+        alert("제출에 성공했습니다");
+        void queryClient.invalidateQueries(["userInfo"]);
+        void queryClient.invalidateQueries(["resume"]);
+      },
```

Comment:
```suggestion
        void queryClient.invalidateQueries(["recruit"]);
      },
```

홈에서 사용하는 recruit 쿼리도 Resume의 제출 여부에 영향을 받으니 invalidate하는 게 좋을 것 같습니다